src/main/resources/messages.properties

```diff
@@ -35,8 +35,8 @@ public.user.activation.title.text=User Activation
 public.user.activation.password.one=Create your password
 public.user.activation.password.two=Repeat password 
 public.user.activation.expired.title=Link Expired
-public.user.activation.expired.text.p1.1=To get a new link, you need to
-public.user.activation.expired.text.p1.2=create your account again
+public.user.activation.expired.text.p1=To get a new link, you need to create your account again.
+public.user.activation.expired.text.p1.hyperlink=To get a new link, you need to <a href="{0}">create your account again</a>.
```

Comment:
As this is for 2.1, these need Welsh translations.
Please also make sure that you rename the messages using refactoring as it seems that you haven't changed the Welsh equivalents.

Reply:
Good point

Reply:
This text will change again in SIDM-3404.